docs/configuration.md

```diff
@@ -327,6 +327,13 @@ By default, the cache dynamically resizes according to the amount of free memory
 The option is set to 0 by default, which enables dynamic resizing. The default on previous versions of
 EventStoreDb was 100,000 entries.
 
+::: note
+The default value of 0 for `StreamInfoCacheCapacity` might not always be the best value for optimal performance. Ideally, it should be set to double the number of streams in the anticipated working set.
+
+The number of streams can be obtained from the `$streams` system projection. Firstly, the `$streams` system projection would need to be enabled. Once it is complete to 100% (which can be verified from the UI on the Projections page under the "Done" column),
```

Comment:
It's a good idea to point out how to get the total stream count, but I think it should be enough to make the suggestion and then link to the projection's documentation. e.g;
```suggestion
The total number of streams can be obtained by checking the event count in the `$streams` system stream. This stream is created by the [$streams system projection](https://developers.eventstore.com/server/v24.2/projections.html#streams-projection).
```

Perhaps also add a disclaimer that the total number of streams does not necessarily give you the anticipated working set. The working set of streams is the set of streams that you intend on actively reading/writing/subscribing to. This can be much lower than the total number of streams in certain cases, especially in systems that have many short-lived streams